pkg/index/job/correction/usecase/corrector.go

```diff
@@ -204,23 +200,25 @@ func (r *run) Start(ctx context.Context) (<-chan error, error) {
 		log.Infof("correction finished in %v", end)
 		return nil
 	}))
-
 	return ech, nil
 }
 
 func (r *run) PreStop(ctx context.Context) error {
-	r.corrector.PreStop(ctx)
-	return nil
+	return r.corrector.PreStop(ctx)
```

Comment:
> **NOTE**
It may be better to return the result of the method so that a caller can properly handle error.